static/js/components/tiles/scatter_tile.tsx

```diff
@@ -115,15 +123,17 @@ export function ScatterTile(props: ScatterTilePropType): JSX.Element {
   const [scatterChartData, setScatterChartData] = useState<
     ScatterChartData | undefined
   >(null);
-  const [isLoading, setIsLoading] = useState(false);
-
+  const [isLoading, setIsLoading] = useState(true);
+  const { shouldLoad, containerRef } = useLazyLoad(props.lazyLoadMargin);
   useEffect(() => {
+    if (props.lazyLoad && !shouldLoad) {
+      return;
+    }
     if (scatterChartData && areDataPropsEqual()) {
       // only re-fetch if the props that affect data fetch are not equal
       return;
     }
     (async () => {
-      setIsLoading(true);
```

Comment:
curious about no longer setting loading before fetching data .. I think this might affect the loading behavior in the vis tool when options are changed?

here & elsewhere

Reply:
Good point- added setIsLoading back here